### PR TITLE
Remove info log of normalizing query while searching with cosine metric

### PR DESCRIPTION
**Pull request by chasingegg**
`Normalize` will be called at both `Train` and `Search` stage, which will generate many info logs while searching with cosine metric.

### CODE REVIEW

File: src/common/utils.cc [hunk 1]
```diff
@@ -50,8 +50,6 @@ Normalize(const DataSet& dataset) {
     auto dim = dataset.GetDim();
     float* data = (float*)dataset.GetTensor();
 
-    LOG_KNOWHERE_INFO_ << "vector normalize, rows " << rows << ", dim " << dim;
```
Comment (cydrain) [inline src/common/utils.cc:53]: you can change the log level from INFO to DEBUG

Comment (chasingegg) [reply src/common/utils.cc:53]: Done.

**Pull request merged**